### DISCUSSION TITLE
Lightdm

### DISCUSSION THREAD

**Discussion by Alextor98405** [hyprwm/Hyprland#307] [Q&A]
are there any plans on making this kick ass WM work with lightdm.... im on i3 arch and it usues lightdm out of the box. I would like to keep the OG greeter.

**Comment by vaxerski** [comment]
No login managers are supported, and I will not go out of my way to support each and every one. It's their job, not mine. SDDM managed to, and it works flawlessly.

**Comment by Alextor98405** [comment]
I want to do a review on my channel of you wm...I'll change my greeterSent from my T-Mobile 5G Device
-------- Original message --------From: Vaxry ***@***.***> Date: 7/5/22  12:19 AM  (GMT-08:00) To: hyprwm/Hyprland ***@***.***> Cc: Alextor98405 ***@***.***>, Author ***@***.***> Subject: Re: [hyprwm/Hyprland] Lightdm (Discussion #307) 
No login managers are supported, and I will not go out of my way to support each and every one. It's their job, not mine. SDDM managed to, and it works flawlessly.

**Comment by Mrfiregem** [comment]
If you mean that the program doesn't show up, are you sure that the [desktop file](https://github.com/hyprwm/Hyprland/blob/main/example/hyprland.desktop) is in `/usr/share/wayland-sessions`?

**Comment by Alextor98405** [comment]
Ok.. thank.you Sent from my T-Mobile 5G Device
-------- Original message --------From: Vaxry ***@***.***> Date: 7/5/22  12:19 AM  (GMT-08:00) To: hyprwm/Hyprland ***@***.***> Cc: Alextor98405 ***@***.***>, Author ***@***.***> Subject: Re: [hyprwm/Hyprland] Lightdm (Discussion #307) 
No login managers are supported, and I will not go out of my way to support each and every one. It's their job, not mine. SDDM managed to, and it works flawlessly.
